test/project_types/theme/commands/push_test.rb

```diff
@@ -96,11 +97,13 @@ def test_push_with_stable
       end
 
       def test_push_with_root
+        specified_root = ShopifyCLI::ROOT + "/test/fixtures/theme"
```

Comment:
I had to use a different root, because `theme serve push` in a non-existing folder like `dist` does not make sense.